src/prpy/util.py

```diff
@@ -749,3 +749,86 @@ def IsTimedTrajectory(trajectory):
     """
     cspec = trajectory.GetConfigurationSpecification()
     return cspec.ExtractDeltaTime(trajectory.GetWaypoint(0)) is not None
+
+
+def jointposvelacc_from_traj(robot, traj, time):
```

Comment:
Should probably keep with the OpenRAVE naming convention used elsewhere: `JointPosVelAccFromTraj`.

Is there a simpler name for this?  Maybe `JointMotionFromTraj`?